templates/students/update_profile.html

```diff
@@ -149,4 +135,32 @@ <h2>
         }, false)
     }())
 </script>
+
+<!-- Script for Image Validation and auto-submitting the form-->
+<script type="text/javascript">
+    function imageValidation() {
+        var fileInput = document.getElementById('profileImage');
+        var filePath = fileInput.value;
+        var allowedExtensions = /(\.jpg|\.jpeg|\.png|\.gif)$/i;
+        if (!allowedExtensions.exec(filePath)) {
+            alert('Please upload file having extensions .jpeg, .jpg, .png or .gif only.');
+            fileInput.value = '';
+        }
+        else{
+            document.getElementById("form-update-profile").submit();
```

Comment:
Use jQuery and not vanilla JS. Replace it with `$("#student-profile-update-form").submit()`.